src/include/user/rtnr.h

```diff
@@ -26,6 +26,7 @@ struct sof_rtnr_config {
 	uint32_t reserved[4];
 
 	struct sof_rtnr_params params;
+	int32_t data[];
```

Comment:
Wont you need a size here to tell cleint code how big data[] is ?

Reply:
The data is with variable length. Actual size is stored in the size flag.